pilot/pkg/networking/core/tracing.go

```diff
@@ -615,6 +615,36 @@ func buildServiceTags(metadata *model.NodeMetadata, labels map[string]string) []
 				},
 			},
 		},
+		{
```

Comment:
I realize that a) we should be more surgical in what we expose in tags b) we should have this for waypoint only.